tests/Configuration/LoaderTest.php

```diff
@@ -35,4 +36,15 @@ public function testLoadThrowsFileCannotBeParsedAsYamlExceptionWhenFileDoesNotCo
 
         $loader->load($file);
     }
+
+    public function testLoadThrowsParsedYamlIsNotAnArrayExceptionWhenFileDoesNotContainYamlThatCanBeParsedToAnArray(): void
+    {
+        $file = __DIR__.'/Fixtures/does-not-contain-array.yml';
```

Comment:
If you prefer, I can also use lower-case for the `Fixtures` directory.

Reply:
No, that's fine. There are already some `Fixtures` directories.